src/index.css

```diff
@@ -1,26 +1,162 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 :root {
   --title: #0290ff;
   --link: #121212;
+  --panel: #fafafa;
+}
+
+.single-book,
+.category {
+  background: var(--panel);
+  margin: 20px 5%;
+  box-shadow: 8px 8px 25px 0 rgba(0, 0, 0, 0.5);
+}
```

Comment:
- [ ] You have missed this two-part `update progress` and `completed`. Kindly implement it
### Design
![figma](https://user-images.githubusercontent.com/81727988/175799683-4e89d926-76d9-4df8-bcbc-b1adab3a448b.png)